src/spin_sdk/mqtt.py

```diff
@@ -0,0 +1,18 @@
+"""Module for utilizing Spin Outbound MQTT"""
+
+from enum import Enum
+from spin_sdk.wit.imports.mqtt import Connection, Qos as Qos 
+
+def open(address: str, username: str, password: str, keep_alive_interval_in_secs: int) -> Connection:
+    """
+    Open a connection to the Mqtt instance at `address`.
+    
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.redis.ErrorInvalidAddress)` will be raised if the connection string is invalid.
+
+    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.redis.ErrorTooManyConnection)` will be raised if there are too many open connections. Closing one or more previously opened connection using the `__exit__` method might help.
```

Comment:
```suggestion
    A `spin_sdk.wit.types.Err(spin_sdk.wit.imports.redis.ErrorTooManyConnections)` will be raised if there are too many open connections. Closing one or more previously opened connection using the `__exit__` method might help.
```

`pdoc` should be printing warnings when names like this are misspelled.  Please try running scripts/generate_docs.py again and pay close attention to any warnings.